tests/schema/__snapshots__/filtering-ordering.test.ts.snap

```diff
@@ -1,12 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`in dev stage, removes filtering or ordering entirely if no arg or wrong args are passed and log error: output 1`] = `
-"
-Warning: Your GraphQL \`Query\` object definition is projecting a relational field \`users\`. On it, you are declaring that clients be able to filter by Prisma \`User\` model field \`somethingWrong\`. However, your Prisma model \`User\` model has no such field \`somethingWrong\`
-
-Warning: Your GraphQL \`Query\` object definition is projecting a relational field \`users\`. On it, you are declaring that clients be able to order by Prisma \`User\` model field \`somethingWrong\`. However, your Prisma model \`User\` model has no such field \`somethingWrong\`
-"
-`;
+exports[`in dev stage, removes filtering or ordering entirely if no arg or wrong args are passed and log error: output 1`] = `""`;
```

Comment:
Broken proxy here as well

Reply:
Seems to be a result of the same incorrect merge conflict resolution as the other case. It looks like it's been resolved as of my latest update, but I don't know much about this functionality so would encourage you to take a look.